components/core/SearchBar/SearchBar.test.js

```diff
@@ -0,0 +1,47 @@
+import { fireEvent, stubbedRender } from '@/testUtils';
```

Comment:
SearchBar doesn't have 100% coverage, would be nice if we could reach that for this one too :)